src/axom/sidre/core/Group.hpp

```diff
@@ -282,6 +282,47 @@ class Group
   }
 #endif
 
+  /*!
+   * \brief Return information about data associated with Group subtree with
+   *        this Group at root of tree (default 'recursive' is true), or for 
+   *        this Group only ('recursive' is false) in fields of given 
+   *        Conduit Node.
```

Comment:
Same comment as before about usage of "Return"

Reply:
Got it. I will change.